src/redux/store.js

```diff
@@ -0,0 +1,72 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { getTodos, getUsers } from '../api/getData';
+
+const START_LOAD = 'start_loading';
```

Comment:
назву action прийнято писати також в uppercase
```suggestion
const START_LOAD = 'START_LOAD';
```